cloud/amazon/ec2_lc.py

```diff
@@ -212,6 +221,15 @@ def create_launch_config(connection, module):
         classic_link_vpc_id=classic_link_vpc_id,
     )
 
+    if append_hash:
+        # MD5 of launch configuration properties
+        h = hashlib.md5()
+        h.update(str(frozenset(lc.__dict__)))
+
+        # Update name variables with md5 hash
+        name = '-'.join((name, h.hexdigest()))
```

Comment:
@ryansb thanks for the feedback.  I think aligning with API capabilities and lower probability of collisions makes sense for automation.  Is there a specific operation in the console this complicates?  I've been using it in production for some time and haven't hit any issues.

My vote is for more bits.  According to https://en.wikipedia.org/wiki/Birthday_attack 32 bit collisions would creep into the realm of possibility without exceeding launch configuration limits.  It probably won't happen to either of us but why make it more likely?